app/routes/[_]actions.newsletter.tsx

```diff
@@ -1,7 +1,7 @@
 import { data } from "react-router";
 import { subscribeToNewsletter } from "~/lib/convertkit";
 import { requirePost } from "~/lib/http.server";
-import type { Route, Info } from "./+types/[_]actions.newsletter";
+import type { Route } from "./+types/[_]actions.newsletter";
```

Comment:
I guess we stopped exporting `Info`?  I changed the fetchers that were using this to just use `typeof action`